docs/docs/glossary/headless-wordpress.md

```diff
@@ -7,7 +7,7 @@ Learn what <q>headless WordPress</q> means, how it differs from other ways of us
 
 ## What is Headless WordPress?
 
-_Headless WordPress_ is the practice of using WordPress as a [headless CMS](/docs/how-to/sourcing-data/headless-cms/) for your [JAMstack](/docs/glossary/jamstack/) site. Headless WordPress takes advantage of the WordPress REST API to separate its content from the frontend that displays it.
+_Headless WordPress_ is the usage of the content editing and management capabilities of WordPress, decoupled from the front-end templating features of the CMS.
```

Comment:
Trying to explain this in a slightly simpler way. Also pulled in the paragraph from the end, bc I think the context is more helpful in this introduction

```suggestion
A _headless WordPress_ site is one that uses WordPress for managing content and some other custom frontend stack to display that content. Headless WordPress enables content writers to use a familiar interface while giving web developers the flexibility to use any frontend technology stack.
```